src/BibleWell.App/App.axaml.cs

```diff
@@ -190,6 +195,21 @@ private void ConfigureUserPreferences(IUserPreferencesService userPreferencesSer
             userPreferencesService.Remove(PreferenceKeys.Language);
         }
     }
+    
+    private void ConfigurePushNotifications()
+    {
+        // Subscribe to push notification actions
+        if (_pushNotificationActionService != null)
+        {
+            _pushNotificationActionService.ActionTriggered -= NotificationActionTriggered;
+        }
+        
+        _pushNotificationActionService = Ioc.Default.GetService<IPushNotificationActionService>();
+        if (_pushNotificationActionService != null)
+        {
+            _pushNotificationActionService.ActionTriggered += NotificationActionTriggered;
```

Comment:
Instead of wiring up an event here in this `App` class, could the `pushNotificationActionService` instead hold the logic for acting on a push notification?

Right now it's just routing based upon the actoin so it seems like that could be self-contained in the `PushNotificationActionService`.  If so, it remove the need for all of these additions to the App.axaml.cs.